tools/logger/convert.c

```diff
@@ -198,6 +198,12 @@ static void process_params(struct proc_ldc_entry *pe,
 		if (p[1] == '%') {
 			/* Skip "%%" */
 			p += 2;
+		} else if (p[1] == 's') {
+			/* check for string printing, because it leads to logger crash */
+			log_err("String printing is not supported\n");
```

Comment:
Is there anyway we can print the line and file where the trace is attempting a string (to help the developer fix).

Reply:
Full message (including trace file:line location in source code) is printed.